src/Common/src/Interop/Windows/sspicli/SSPIWrapper.cs

```diff
@@ -226,15 +213,17 @@ private enum OP
             VerifySignature
         }
 
-        private static unsafe int EncryptDecryptHelper(OP op, SSPIInterface secModule, SafeDeleteContext context, SecurityBuffer[] input, uint sequenceNumber)
+        private static unsafe int EncryptDecryptHelper(OP op, SSPIInterface secModule, SafeDeleteContext context, Span<SecurityBuffer> input, uint sequenceNumber)
         {
             Interop.SspiCli.SecBufferDesc sdcInOut = new Interop.SspiCli.SecBufferDesc(input.Length);
-            var unmanagedBuffer = new Interop.SspiCli.SecBuffer[input.Length];
+            Span<Interop.SspiCli.SecBuffer> unmanagedBuffer = stackalloc Interop.SspiCli.SecBuffer[input.Length];
+            unmanagedBuffer.Clear();
```

Comment:
Is this because this project has that property turned on that doesn't 0 out stackalloc spans?

Reply:
Yes.